src/styles/blocks/card.scss

```diff
@@ -0,0 +1,97 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
```

Comment:
Would be great to use variables for the most important values. (sizes, colors, etc)
![image](https://user-images.githubusercontent.com/96409563/221639930-defa155d-10be-4c22-8732-22648cce0f7d.png)